marketplacetests/marketplace/pages/base.py

```diff
@@ -36,11 +35,13 @@ def login(self, username, password):
         self.wait_for_login_success_notification()
 
     @property
-    def first_free_app_name(self):
+    def first_free_app(self):
         free_apps = self.search(':free').search_results
-        name = free_apps[0].name
-        self.tap_back()
-        return name
+        app = free_apps[0]
+        name = app.name
+        author = app.author
+        self.tap_home()
+        return {'name': name, 'author': author}
```

Comment:
I'm guessing we're not returning the app object because the properties will be accessed when it's no longer available? I don't think we need to have `name` and `author` variables, we could just: `return {'name': app.name, 'author': app.author}`. Or, if this is all we're returning we could return multiple values: `return app.name, app.author`. The former has better structure and readability though. \</rambling>